sysutils/duplicity/Portfile

```diff
@@ -1,41 +1,40 @@
-# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*-
```

Comment:
Please retain the [standard modeline](https://guide.macports.org/#development.creating-portfile).